components/director/internal/tenantfetcher/README.md

```diff
@@ -0,0 +1,103 @@
+# Tenant Fetcher
+
+## Overview
+
+Tenant Fetcher fetches information about tenants from external APIs.
+
+## Details
+
+This section describes the API schema that a server must implement to integrate with the Tenant Fetcher.
+
+### Authorization
+
+Tenant Events API should use the OAuth 2.0 client credentials authorization flow.
+
+### Endpoints
+
+There are three types of supported endpoints that receive different type of events:
+- Tenant creation endpoint
+- Tenant deletion endpoint
+- Tenant update endpoint
+
+Every endpoints must return a specific payload and accept the following query parameters:
+- **ts** - specifies a timestamp in Unix time, that is the date since events are fetched
+- **page** - specifies the number of page to be fetched, starting from `1`
+- **resultsPerPage** - specifies the number of results included on a single page
+
+#### Tenant creation endpoint
+
+The endpoint should return the following JSON payload:
```

Comment:
`Should`? So what happens if it doesn't?
```suggestion
On success, the endpoint returns the following JSON payload:
```
Or just `The endpoint returns the following JSON payload:` if the payload is returned regardless of the operation result.

See the [guidelines](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/08-style-and-terminology.md#terminology).